README.md

```diff
@@ -44,7 +44,7 @@ When set to true, all tests in the `tests` directory will be run using the Jest
 ### Single step action
 ```yaml
 - name: taqueria tasks
-    uses: ecadlabs/taqueria-github-action@v0.7.0
```

Comment:
Since the action is under active development would it be easier to always reference `ecadlabs/taqueria-github-action@main`  instead of having to specify the Action version?

Reply:
This is just an example of how the user can add the action into their workflow. I think it is better to encourage the user to pin to a specific version to prevent them from experiencing any breaking changes when the action is updated?